polaris-react/src/components/IndexTable/IndexTable.tsx

```diff
@@ -67,14 +67,15 @@ function IndexTableBase({
   emptyState,
   sort,
   lastColumnSticky = false,
+  ...props
 }: IndexTableBaseProps) {
   const {
     loading,
     bulkSelectState,
     resourceName,
     bulkActionsAccessibilityLabel,
-    selectable,
     selectMode,
+    selectable = props.selectable,
```

Comment:
```suggestion
    selectable = restProps.selectable,
```
(non-blocking) If moving forward with the above suggestion